packages/jaeger-ui/src/components/TracePage/TraceTimelineViewer/VirtualizedTraceView.test.js

```diff
@@ -430,4 +431,22 @@ describe('<VirtualizedTraceViewImpl>', () => {
       expect(focusUiFindMatchesMock).toHaveBeenLastCalledWith(trace, spanName, false);
     });
   });
+
+  describe('linksGetter()', () => {
+    let getLinksSpy;
+
+    beforeAll(() => {
+      getLinksSpy = jest.spyOn(getLinks, 'default');
+    });
+
+    afterAll(() => {
+      getLinksSpy.mockRestore();
+    });
+
+    it('calls getLinks with expected params', () => {
+      const span = trace.spans[1];
+      instance.linksGetter(span, span.tags, 0);
+      expect(getLinksSpy).toHaveBeenCalledWith(span, span.tags, 0, trace);
```

Comment:
it would be better to validate the actual value returned, which, as I understand, is `[{url, text}]`

Reply:
I didn't find a clean way to initialize `linkPatterns` for tests, so I exported the `const processedLinks` in `link-patterns.tsx`.